### PR TITLE
Move more documentation to Flex Docs

### PR DESCRIPTION
**Pull request by kpuputti**
- [x] customization checklist
- [x] styling
- [x] icons
- [x] translations
- [x] tos and privacy policy
- [x] static pages
- [x] ci
- [x] deploying to production
- [x] extend listing data
- [x] extended data (removed)
- [x] search filters
- [x] folder structure
- [x] improving performance
- [x] redux
- [x] routing
- [x] update changelog

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -23,12 +23,103 @@ Marketplace features included in Saunatime:
...
+    ├── index.js
+    ├── marketplace-custom-config.js
+    ├── marketplace.css
+    ├── marketplaceFonts.css
+    ├── marketplaceIndex.css
+    ├── reducers.js
+    ├── routeConfiguration.js
+    ├── store.js
+    ├── translations
+    └── util
```
Comment (Gnito) [inline docs/README.md]: A small suggestion, this might be more understandable if directories would include a trailing slash `util/`.

Comment (kpuputti) [reply docs/README.md]: I just copied the content from the existing doc, but you are right. I can change it here.

Comment (kpuputti) [reply docs/README.md]: Fixed.

File: docs/README.md [hunk 2]
```diff
@@ -23,12 +23,103 @@ Marketplace features included in Saunatime:
...
+  (Node.js framework).
+- `server/csp.js` [Content Security Policy](https://content-security-policy.com), whitelist of
+  domains that the application and loaded scripts are allowed to access from the browser.
+- `server/sitemap.js` generates a minimal sitemap.xml for SEO purposes.
+
+### ext/default-mail-templates/
+
+This file contains default email templates for various needs (e.g. _booking-request-accepted_, or
+_verify-your-email_). You should customize all the different files (\*-html.html, \*-subject.txt,
+and \*-text.txt).
```
Comment (Gnito) [inline docs/README.md]: This is outdated stuff, I think. 
```markdown
This file contains default email templates for transaction-specific needs (e.g. _booking-request-accepted_). However, there are some built-in email templates that you can change from [Flex Console](http://flex-console.sharetribe.com/email-templates/email-changed).
```

Comment (kpuputti) [reply docs/README.md]: Fixed.

**Pull request merged**